releasenotes/releasenotes_V0.9.1.md

```diff
@@ -0,0 +1,62 @@
+# Release Notes 0.9.1
+
+
+---
+
+This is a bug fix release for Keptn 0.9.0, containing fixes for the Keptn services.
```

Comment:
Please change:
This is a bug fix release for Keptn 0.9.0, containing fixes for the Keptn services.

to:
This is a bug fix release for Keptn 0.9.0, containing fixes for the Bridge and Keptn core services; especially, lighthouse-service and shipyard-controller.